strawberry/contrib/django/tests/test_model_type.py

```diff
@@ -0,0 +1,167 @@
+from typing import List
+
+import pytest
+
+import strawberry
+from strawberry.contrib.django.tests.models import TestModel
+from strawberry.contrib.django.type import model_type
+from strawberry.graphql import execute
+
+
+@pytest.mark.asyncio
```

Comment:
do we need to test this with async?

Reply:
I believe so, since it uses `async`/`await`. Otherwise pytest skips it.

Reply:
we have graphql sync we can use ;) https://github.com/strawberry-graphql/strawberry/blob/master/tests/test_schema.py#L9